### PR TITLE
Update documentation for render_book().

### PR DESCRIPTION
**Pull request by jonathan-g**
Add details about how the options are set for the output_format parameter to documentation for render_book() function. See discussion in comments to #930.

### CODE REVIEW

**Comment by cderv** [comment]
Thanks ! 
What about adding examples as in the issue, in `\dontrun` clauses maybe ? one with the function & parameters and one with with the sring ? Could be best to prevent convention in the future, right ?

File: NEWS.md [hunk 1]
```diff
@@ -7,6 +7,7 @@
 ## MINOR CHANGES
 
 - The `--file-scope` behavior introduced in bookdown v0.20 is now disabled by default. This is due to broken TOC links for duplicate section names (e.g., "Exercises"; see #909) that have automatically generated identifiers.
+- Updated documentation for `render_book` to make it clearer how options are set for the `output_format` parameter (See #930).
```
Comment (cderv) [inline NEWS.md:10]: As below, syntax is 
```md
(thanks, @yourname, #prnumber)
```
Thanks!

**Comment by jonathan-g** [comment]
I think @yihui makes sense in saying that the documentation fits better into rmarkdown, but it would be useful to put a modicum of documentation into bookdown alerting people to look at the rmarkdown documentation.

If you all want, I can write up some small PRs, one for rmarkdown documenting this behavior and the other for bookdown alerting the user to go look in rmarkdown for details.

I'll defer to what you all would like.

**Comment by cderv** [comment]
Yeah it is right. This should be in `rmarkdown::render`. The help page of `render_book` already linked to `rmarkdown::render`, so having only this documented there seems ok to me. In `render_book` I would maybe only add some example with comments

```r
# will use the default format defined in _output.yml
bookdown::render_book("index.Rmd")
# will use the options for format defines in yaml
bookdown::render_book("index.Rmd",  "pdf_book")
 # If you pass an output format, it must have all the option set
bookdown::render_book("index.Rmd", bookdown::pdf_book(toc = FALSE))
```

and I would add the link to https://bookdown.org/yihui/bookdown/build-the-book.html in a reference part of this help page. 

We could also have `bookdown::render_book` inherits param for `rmarkdown::render` so that both are up to date, but it is not the practice here it seems - we just link to the other topic instead of inheriting.

**Comment by jonathan-g** [comment]
Closed this and created a new PR  #933 following the suggestions from the discussion above. I also created a PR in rmarkdown (https://github.com/rstudio/rmarkdown/pull/1880)

**Pull request closed without merging**